docs/rules/order.md

```diff
@@ -80,35 +80,35 @@ var path = require('path');
 Unbound imports are assumed to have side effects, and will never be moved/reordered. This can cause other imports to get "stuck" around them, and the fix to fail.
 
 ```javascript
-import b from 'b'
-import 'format.css';  // This will prevent --fix from working.
-import a from 'a'
+import b from 'b';
+import 'format.css'; // This will prevent --fix from working.
+import a from 'a';
 ```
 
 As a workaround, move unbound imports to be entirely above or below bound ones.
 
 ```javascript
-import 'format1.css';  // OK
-import b from 'b'
-import a from 'a'
-import 'format2.css';  // OK
+import 'format1.css'; // OK
+import b from 'b';
+import a from 'a';
+import 'format2.css'; // OK
 ```
 
 ## Options
 
 This rule supports the following options (none of which are required):
 
- - [`groups`][18]
- - [`pathGroups`][8]
- - [`pathGroupsExcludedImportTypes`][9]
- - [`distinctGroup`][32]
- - [`newlines-between`][20]
- - [`alphabetize`][30]
- - [`named`][33]
- - [`warnOnUnassignedImports`][5]
- - [`sortTypesAmongThemselves`][7]
- - [`newlines-between-types`][27]
- - [`consolidateIslands`][25]
+- [`groups`][18]
+- [`pathGroups`][8]
+- [`pathGroupsExcludedImportTypes`][9]
+- [`distinctGroup`][32]
+- [`newlines-between`][20]
+- [`alphabetize`][30]
+- [`named`][33]
+- [`warnOnUnassignedImports`][5]
+- [`sortTypesGroup`][7]
+- [`newlines-between-types`][27]
+- [`consolidateIslands`][25]
```

Comment:
```suggestion
 - [`groups`][18]
 - [`pathGroups`][8]
 - [`pathGroupsExcludedImportTypes`][9]
 - [`distinctGroup`][32]
 - [`newlines-between`][20]
 - [`alphabetize`][30]
 - [`named`][33]
 - [`warnOnUnassignedImports`][5]
 - [`sortTypesGroup`][7]
 - [`newlines-between-types`][27]
 - [`consolidateIslands`][25]
```
this seems like an unrelated formatting change

Reply:
Yep yep, I pushed the fix for this and a few other stealth formatting issues  right as you posted this 😄.